python-sdk/nuimages/nuimages.py

```diff
@@ -178,8 +174,8 @@ def shortcut(self, src_table: str, tgt_table: str, src_token: str) -> Dict[str,
         :return: The entry of the destination table correspondings to the source token.
```

Comment:
The entry of the destination table ~correspondings~ corresponding to the source token.